projects/aca-content/folder-rules/src/rule-details/actions/rule-action.ui-component.spec.ts

```diff
@@ -63,7 +63,8 @@ describe('RuleActionUiComponent', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [CoreTestingModule, RuleActionUiComponent]
+      imports: [CoreTestingModule, RuleActionUiComponent],
+      providers: [{ provide: AlfrescoApiService, useClass: AlfrescoApiServiceMock }]
```

Comment:
I believe in every place you have `CoreTestingModule` this line is not needed... we can cleanup the tests even before the refactoring, so that it takes less changes in the future

Reply:
We moved AlfrescoApiService from core to @alfresco/adf-content-service in ADF